src/SickSafetyscannersRos.cpp

```diff
@@ -382,7 +382,20 @@ SickSafetyscannersRos::createLaserScanMessage(const sick::datastructure::Data& d
 {
   sensor_msgs::LaserScan scan;
   scan.header.frame_id = m_frame_id;
-  scan.header.stamp    = ros::Time::now();
+
+  // ros::Time ros_time = ros::Time::now();
```

Comment:
Why leave this commented lines here?

Reply:
I wanted to leave them there so that we don't have to rewrite them in case we need to debug the driver in the future. But they are now removed.